lib/pbio/test/src/uartdev.c

```diff
@@ -626,7 +626,7 @@ static PT_THREAD(test_boost_interactive_motor(struct pt *pt)) {
 }
 
 static PT_THREAD(test_technic_large_motor(struct pt *pt)) {
-    // info messages captured from Technic Large Linear Motor with logic analyzer
+    // info messages captured from Technic Control+ Large Linear Motor with logic analyzer
```

Comment:
I think Control+ is just the name of the app, not the motors or hubs. e.g. https://www.lego.com/en-us/product/technic-large-motor-88013 doesn't say Control+.

Reply:
Ok, I edited the commit accordingly.